recipes/llama-cpp/all/conandata.yml

```diff
@@ -1,17 +1,17 @@
 sources:
+  "b4570":
+    url: "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b4570.tar.gz"
+    sha256: "35bfe07807fd0cf30710023765b9a7ab6c1003f27ef907ce9cea2c5464411430"
   "b4079":
     url: "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b4079.tar.gz"
     sha256: "79093413dcdbd30f83b800aeb958c87369fdfdaf4e5603b094185898ff404a32"
   "b3542":
     url: "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b3542.tar.gz"
     sha256: "6f8b23d930400fce5708d2c85022ef33f1083af8f6ac395abefadacee0942e78"
-  "b3040":
-    url: "https://github.com/ggerganov/llama.cpp/archive/refs/tags/b3040.tar.gz"
-    sha256: "020e040139660eb40113503bb1057d5387677d249b990e008e04821532f7cd62"
 patches:
+  "b4570":
+    - patch_file: "patches/b4570-001-curl-patch-targets.patch"
   "b4079":
     - patch_file: "patches/b4079-001-curl-patch-targets.patch"
   "b3542":
     - patch_file: "patches/b3542-001-curl-patch-targets.patch"
-  "b3040":
```

Comment:
This version had a slightly different logic, because did not have the ggml component so it the recipe was a bit cleaner removing this, and it's also quiet outdated